sysutils/yabai/files/patch-makefile.diff

```diff
@@ -1,19 +1,19 @@
---- makefile.orig	2022-09-26 10:06:45.000000000 +0200
-+++ makefile	2022-09-26 10:19:48.000000000 +0200
+--- makefile	2024-09-30 00:05:50
++++ makefile.orig	2024-09-30 01:16:00
 @@ -1,7 +1,7 @@
  FRAMEWORK_PATH = -F/System/Library/PrivateFrameworks
  FRAMEWORK      = -framework Carbon -framework Cocoa -framework CoreServices -framework CoreVideo -framework SkyLight
  CLI_FLAGS      =
--BUILD_FLAGS    = -std=c99 -Wall -g -O0 -fvisibility=hidden -mmacosx-version-min=11.0 -fno-objc-arc -arch x86_64 -arch arm64 -sectcreate __TEXT __info_plist $(INFO_PLIST)
-+CFLAGS         += -std=c99 -Wall -DNDEBUG -O3 -fvisibility=hidden -fno-objc-arc -arch x86_64 -arch arm64 -sectcreate __TEXT __info_plist $(INFO_PLIST)
+-BUILD_FLAGS    = -std=c11 -Wall -Wextra -g -O0 -fvisibility=hidden -mmacosx-version-min=11.0 -fno-objc-arc -arch x86_64 -arch arm64 -sectcreate __TEXT __info_plist $(INFO_PLIST)
++CFLAGS         += -std=c11 -Wall -Wextra -DNDEBUG -O3 -fvisibility=hidden -fno-objc-arc -arch x86_64 -arch arm64 -sectcreate __TEXT __info_plist $(INFO_PLIST)
```

Comment:
if this now needs C11 then you should request a compatible compiler in the PortGroup. 

As with "compiler.c_standard 2011".

Reply:
Upstream changed to c11.

Will enact requested change.

Reply:
ping - please address this so we can merge the PR

Reply:
@reneeotten sorry, lots going on and I forgot. Updated.